raster3d/r3.out.netcdf/main.c

```diff
@@ -135,8 +135,14 @@ static void write_netcdf_header(int ncid, RASTER3D_Region *region, int *varid,
     if ((retval = nc_put_att_text(ncid, NC_GLOBAL, "Conventions",
                                   strlen(CF_SUPPORT), CF_SUPPORT)))
         ERR(retval);
+
+    // Works up to version 9999.
+    // 2 in original, 2 more, \0 is in the original string which sizeof counts.
+    size_t history_size = sizeof(HISTORY_TEXT) / sizeof(char) + 2;
+    char history_string[history_size];
```

Comment:
Seems to me a bit overcomplicated.

What about using macros:

```c
#define STR_HELPER(x)         #x
#define STR(x)                STR_HELPER(x)
#define HISTORY_TEXT \
    "GRASS GIS " STR(GRASS_VERSION_MAJOR) " netCDF export of r3.out.netcdf"
```

then you can use `HISTORY_TEXT` as before?

Reply:
Thank you. That's much better!